x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -130,9 +132,12 @@ const FormattedFieldValueComponent: React.FC<{
         className={classNames}
         fieldName={fieldName}
         value={value}
-        tooltipProps={{ position: 'bottom', className: dataGridToolTipOffset }}
+        tooltipProps={
+          isUnifiedDataTable ? undefined : { position: 'bottom', className: dataGridToolTipOffset }
+        }
       />
     );
+    if (isUnifiedDataTable) return date;
     return isDraggable ? (
```

Comment:
you can probably do return `isDraggable && !isUnifiedDataTable ?` here, but just a nit

Reply:
makes sense.. Will do.

Reply:
@michaelolo24 , actually thinking it again, this statement restricts `timestamp` to be a draggable for new table, not sure why. I do not think it is correct. Let me know what you think.